proto/ligato/vpp/l3/l3xc.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package ligato.vpp.l3;
+
+option go_package = "go.ligato.io/vpp-agent/v3/proto/ligato/vpp/l3;vpp_l3";
+
+message L3XC {
+    string interface = 1;
+
+    enum Protocol {
+        IPV4 = 0;
+        IPV6 = 1;
+    }
+    Protocol protocol = 2;
+
+    message Path {
+        string interface = 1;
```

Comment:
Maybe we should consider keeping the naming in line with [Route API](https://github.com/ligato/vpp-agent/blob/master/proto/ligato/vpp/l3/route.proto#L24-L25):
- `outgoing_interface`
- `next_hop_addr`